src/github/repository.ts

```diff
@@ -51,18 +49,25 @@ export const deleteRepositoryWebhookHandler = async (context: WebhookContext, ji
 		jiraHost: jiraClient.baseURL,
 		gitHubInstallationId
 	});
+
+	if (!context.payload.repository?.id) {
+		context.log.warn("Cannot delete repository. Missing repository ID");
+		webhookProcessComplete(context, 500, subscription.jiraHost);
+		return;
+	}
+
 	context.log.info(`Deleting dev info for repo ${context.payload.repository?.id}`);
 
+	const { id: repositoryId } = context.payload.repository;
+
 	try {
 		const jiraResponse = await jiraClient.devinfo.repository.delete(
-			transformRepositoryId(context.payload.repository?.id, context.gitHubAppConfig?.gitHubBaseUrl)
+			repositoryId,  context.gitHubAppConfig?.gitHubBaseUrl
```

Comment:
I LIKE passing in the origin repoId and let the method itself do the transforming.